### DISCUSSION TITLE
species: hvplot fails to find color column

### DISCUSSION THREAD

**Discussion by byandell** [earthlab-education/Earth-Analytics-AY24#574] [Q&A]
Please see
<https://github.com/earthlab-education/species-distribution-coding-challenge-byandell/blob/main/notebooks/sandhill-crane-species-download.ipynb>
in section **Download and save ecoregion boundaries** around chunks 25-29.

I verified that `occurrence_gdf` has column `norm_occurrences`, but the call to `hvplot` gives the message below. This seems to indicate either it does not find this column or it is expecting a different type of column for color. I followed the same layout as @lauren-alexandra in her
[willow-flycatcher-distribution.ipynb](https://github.com/lauren-alexandra/lauren-alexandra.github.io/blob/main/willow-flycatcher-distribution/willow-flycatcher-distribution.ipynb).
I cannot figure out what is wrong.

AbbreviatedException: ValueError: failed to validate MultiPolygons(id='9620438e-dd20-423b-9720-fb26b83afdd0', ...).fill_color: expected an element of either String, Nullable(Color), Instance(Value), Instance(Field), Instance(Expr), Struct(value=Nullable(Color), transform=Instance(Transform)), Struct(field=String, transform=Instance(Transform)) or Struct(expr=Instance(Expression), transform=Instance(Transform)), got dim('norm_occurrences')

**Comment by hanried** [comment]
Hi @byandell! I think I had a similar error and asked about it in this discussion post: https://github.com/earthlab-education/Earth-Analytics-AY24/discussions/562

**Comment by byandell** [comment]
Agreed. @akpost21  helped me with this as well. Key as @eculler points out is to "comment (out) the line with `geo=True`.
